packages/core/test/trader/uniswap-broker/UniswapBroker.js

```diff
@@ -0,0 +1,240 @@
+const { MAX_UINT_VAL } = require("@uma/common");
+const { toWei, toBN, fromWei } = web3.utils;
+const { getTruffleContract } = require("@uma/core");
+
+// Tested Contract
+const UniswapBroker = artifacts.require("UniswapBroker");
+const Token = getTruffleContract("ExpandedERC20", web3);
+const WETH9 = getTruffleContract("WETH9", web3);
+
+// Helper Contracts
+const UniswapV2Factory = require("@uniswap/v2-core/build/UniswapV2Factory.json");
```

Comment:
I was able to work around importing contract byte code by simply pulling in the build artefacts from the packages like this :)

Reply:
Nice

Reply:
Very very nice. +1 @chrismaree